source/globalCommands.py

```diff
@@ -2494,12 +2483,11 @@ def _getScriptNameForProfile(cls, name):
 		name = name.encode("utf-8")

```

Comment:
In theory, yes this may work. When run, this actually fails because isalnum wants a character when in fact it becomes an integer (str.encode(charset) actually gives us an array of ints); I also came across this one while testing this earlier.

Reply:
Wow, nice gotcha there. Let's see how to fix this properly.